packages/system/src/conditional.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { conditional } from './conditional';
+
+test('returns an array of strings', () => {
```

Comment:
I think this test is more confusing than helping. remove it, the next test already tests this too